src/components/calendar/calendar.ts

```diff
@@ -665,15 +664,11 @@ export default class IgcCalendarComponent extends EventEmitterMixin<
   protected override render() {
     const direction = this._isDayView && this.orientation === 'horizontal';
 
-    const styles = {
-      display: 'flex',
-      flexGrow: 1,
-      flexDirection: direction ? 'row' : 'column',
-    };
+    const part = direction ? 'content' : 'content content-vertical';
```

Comment:
```suggestion
    const parts = {
      content: true,
      'content-vertical': this._isDayView && this.orientation === 'vertical',
    };
```
You can also delete the `direction` variable above it.